dcos/package.py

```diff
@@ -58,10 +58,13 @@ def install_app(pkg, revision, init_client, options, app_id):
     init_desc = pkg.marathon_json(revision, options)
 
     if app_id is not None:
-        logger.debug('Setting app ID to "%s" (was "%s")',
-                     app_id,
-                     init_desc['id'])
-        init_desc['id'] = app_id
+        if not isinstance(init_desc, list):
+            logger.debug('Setting app ID to "%s" (was "%s")',
+                         app_id,
+                         init_desc['id'])
+            init_desc['id'] = app_id
+        else:
+            logger.debug('Not setting app ID for multi app json')
```

Comment:
The other option is to append -1, -2, etc to the app-id name, but that is also arbitrary. I think its fine to leave as is.